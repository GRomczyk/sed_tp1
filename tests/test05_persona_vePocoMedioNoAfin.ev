00:00:01:0 in_port1 [-0.5,1.0]
00:00:02:0 info 1
00:00:03:0 in_port1 [-0.5,1.0]
00:00:04:0 info 1
00:00:05:0 in_port1 [-0.5,1.0]
00:00:06:0 info 1
00:00:07:0 in_port1 [-0.5,1.0]
00:00:08:0 info 1
00:00:09:0 in_port1 [-0.5,1.0]
00:00:10:0 info 1
00:00:11:0 in_port1 [-0.5,1.0]
00:00:12:0 info 1
00:00:13:0 in_port1 [-0.5,1.0]
00:00:14:0 info 1
00:00:15:0 in_port1 [-0.5,1.0]
00:00:16:0 info 1
00:00:17:0 in_port1 [-0.5,1.0]
00:00:18:0 info 1
00:00:19:0 in_port1 [-0.5,1.0]
00:00:20:0 info 1
00:00:21:0 in_port1 [-0.5,1.0]
00:00:22:0 info 1
00:00:23:0 in_port1 [-0.5,1.0]
00:00:24:0 info 1

