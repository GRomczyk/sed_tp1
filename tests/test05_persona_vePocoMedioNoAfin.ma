[Top]
%en este test esperamos que la frecuencia con la que la persona ve una %noticia, dependa de su afinidad
components : P1@Persona

out : info_out1
in : in_port1 info

Link : in_port1 inMedios@P1
Link : info infoRequest@P1

Link : outInfo@P1 info_out1

[P1]
p : 1.0
influenciabilidad : 1.0
poder_convencimiento : 0.2
tiempo_sillon : 1.0
grado : 1
periodo_medio : 60000000
