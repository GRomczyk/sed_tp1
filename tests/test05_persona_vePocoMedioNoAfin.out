00:00:02:000:0 info_out1      1.00000
00:00:04:000:0 info_out1      1.00000
00:00:06:000:0 info_out1      1.00000
00:00:08:000:0 info_out1      1.00000
00:00:10:000:0 info_out1      0.71016
00:00:12:000:0 info_out1      0.71016
00:00:14:000:0 info_out1      0.13926
00:00:16:000:0 info_out1     -0.43940
00:00:18:000:0 info_out1     -0.49999
00:00:20:000:0 info_out1     -0.50000
00:00:22:000:0 info_out1     -0.50000
00:00:24:000:0 info_out1     -0.50000
